h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var counter = element.parentElement.querySelector(
+    '.js-character-limit-counter');
```

Comment:
I renamed this so that all of the CSS classes that are only meant to be used internally by `CharacterLimitController` are `"js-character-limit-*"`.